### DISCUSSION TITLE
v0.5.0 in PyPI

### DISCUSSION THREAD

**Discussion by ashrude** [a-parida12/pdf2dcm#61] [General]
Hi, I was just wondering when version 0.5.0 was going to be released on PyPI? I'm not too familiar with how pip/PyPI work

**Comment by a-parida12** [comment]
@ashrude Thanks for pointing out that the pypi version is behind. I will look into it and update the pypi version to 0.5.0

**Comment by a-parida12** [comment]
The version 0.5.0 is now available for download from pypi.
